bazel/api-gen/rendering/templates/deprecated-label.tsx

```diff
@@ -21,7 +21,7 @@ export function DeprecatedLabel(props: {
         <span dangerouslySetInnerHTML={{__html: entry.deprecationMessage}}></span>
       </div>
     );
-  } else if ('isDeprecated' in entry) {
+  } else if ('isDeprecated' in entry && entry.isDeprecated) {
```

Comment:
Shouldn't we just do `entry.isDeprecated`?

If it doesn't exist then it will be falsy anyway

Reply:
This is due to `entry` being a union that we need to discriminate.

Reply:
👍